rootfs/etc/nginx/lua/lua_ingress.lua

```diff
@@ -2,25 +2,40 @@ local _M = {}
 
 local seeds = {}
 local original_randomseed = math.randomseed
-math.randomseed = function(seed)
-  local pid = ngx.worker.pid()
 
-  if seeds[pid] then
+local function get_seed_from_urandom()
```

Comment:
Can you explain the rationale behind this function implementation? What's the entropy going to be with this implementation?

Why not use rand_bytes from OpenSSL?

Reply:
`/dev/urandom` provides an interface to the kernel's random number generator.

Reply:
> 4 bytes is good enough for us.

How do you decide that? Why not i.e 8 bytes?

Reply:
> > 4 bytes is good enough for us.
> 
> How do you decide that? Why not i.e 8 bytes?

`random` in this repo is only for choose balancer, not for encryption, so we don't need to read  more bytes. If for encryption, we need read 8 bytes maybe.